features/minion/heartbeat/consumer/src/main/java/org/opennms/minion/heartbeat/consumer/HeartbeatConsumer.java

```diff
@@ -154,7 +154,7 @@ private void provision(final OnmsMinion minion,
             return;
         }
 
-        // Return fast until the provisioner is running to pick up the events send below
+        // Return fast until the provisioner is running to pick up the events sent below
         if (!this.eventSubscriptionService.hasEventListener(EventConstants.RELOAD_IMPORT_UEI)) {
```

Comment:
I think we should really change the ```component-dao.xml``` into a context that is loaded later (like a ```component-service.xml```) since it relies on the EventIpcManager. BTW, I grepped and I think that this is the only ```component-dao.xml``` where we're injecting/autowiring the EventIpcManager interfaces so I don't there are similar problems in any other code.